api/policies/buildCallback.js

```diff
@@ -3,7 +3,7 @@
  * Requests must have a `token` param that matches config and an `id`
  */
 module.exports = function (req, res, next) {
-  if (req.param('token') !== sails.config.build.token || !req.param('id')) {
```

Comment:
Now the above comment is incorrect--- which is right, the comment or the code?

Code wins. Flawless victory. Might I suggest removal of the entire comment?